core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -176,7 +176,36 @@ object EntityDecoder extends EntityDecoderInstances {
     override val consumes: Set[MediaRange] = (r1 +: rs).toSet
   }
 
-  /** Helper method which simply gathers the body into a single ByteVector */
+  /**
+  private class OrDec[F[_]: Functor, T](a: EntityDecoder[F, T], b: EntityDecoder[F, T])
```

Comment:
So this is unused now...

Reply:
Oh, neat, because `orElse` is a SemigroupK now.  (I wonder what changed in this merge, though.)